lib/client.js

```diff
@@ -128,7 +128,13 @@ Client.prototype.request = function (method, path, data) {
             }
             resolve(res);
         });
-    }).bind(this);
+    }).bind(this)
```

Comment:
I'm not sure that `beam-client-node` is actually used in the frontend at the moment. That may change in the future as we move off wsabi